### PR TITLE
test: add cron job for pos e2e tests

### PR DESCRIPTION
**Pull request by gjermundgaraba**
Added a cron job to run once per day (midnight) with e2e tests with PoS.

Also added a way to run the tests manually with manual dispatch (and select the pow or pos): https://github.com/cosmos/solidity-ibc-eureka/actions/runs/11629265273

### CODE REVIEW

File: .github/workflows/e2e.yml [hunk 1]
```diff
@@ -94,6 +106,17 @@ jobs:
...
+        if: env.ETH_TESTNET_TYPE == 'pos'
+        run: |
+          echo "deb [trusted=yes] https://apt.fury.io/kurtosis-tech/ /" | sudo tee /etc/apt/sources.list.d/kurtosis.list
+          sudo apt update
+          sudo apt install kurtosis-cli=1.3.0 # Must match the kurtosis library version we use in the e2e tests
+          kurtosis engine start
+          kurtosis analytics disable
+          echo "$(dirname $(which kurtosis))" >> $GITHUB_PATH
+        shell: bash
+
```
Comment (srdtrk) [inline .github/workflows/e2e.yml:119]: There isn't an action to setup kurtosis?

File: e2e/interchaintestv8/e2esuite/light_clients.go [hunk 2]
```diff
@@ -70,8 +70,18 @@ func (s *TestSuite) UpdateEthClient(ctx context.Context, ibcContractAddress stri
...
+		for _, update := range lightClientUpdates {
+			if update.Data.AttestedHeader.Beacon.Slot > highestUpdateSlot {
+				highestUpdateSlot = update.Data.AttestedHeader.Beacon.Slot
+			}
+		}
+
 		return finalityUpdate.Data.FinalizedHeader.Beacon.Slot > uint64(updateTo) &&
-				targetPeriod > trustedPeriod,
+				targetPeriod > trustedPeriod &&
+				highestUpdateSlot > uint64(updateTo),
```
Comment (srdtrk) [inline e2e/interchaintestv8/e2esuite/light_clients.go:84]: Maybe add some comments explaining what's going on in here?

**Review by srdtrk — APPROVED** [review]
Lgtm. Just added a few comments

File: .github/workflows/e2e.yml (hunk 1 above)
Comment (gjermundgaraba) [reply .github/workflows/e2e.yml:119]: No, the only one I found also runs a lot of tests and other things we don't want

**Pull request merged**